gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/GremlinDriverIntegrateTest.java

```diff
@@ -64,13 +64,7 @@
 import java.awt.Color;
 import java.io.File;
 import java.time.Instant;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
thanks for the test. it looks like your IDE auto-formatted the imports. please note that we don't use wildcard imports in our code style. could you please revert that change?

Reply:
Ah you're right, when I added `import java.util.UUID;`, IntelliJ automatically squashed the imports. Reverted it. Thanks !